velox/functions/prestosql/json/SIMDJsonUtil.h

```diff
@@ -47,4 +47,9 @@ void simdjsonErrorsToExceptions(
 simdjson::simdjson_result<simdjson::ondemand::document> simdjsonParse(
     const simdjson::padded_string_view& json);
 
+/// Parse the input incomplete json string using a thread local on demand
```

Comment:
/// Parse the input json string using a thread local on demand parser. Allow incomplete json input.

Reply:
Done